filebeat/filebeat.yml

```diff
@@ -453,3 +453,15 @@ logging.files:
 
   # Number of rotated log files to keep. Oldest files will be deleted first.
   #keepfiles: 7
+
+#================================ Filters =====================================
+
+# This section defines a list of filtering rules that are applied one by one starting with the
+# exported event:
+#   event -> filter1 -> event1 -> filter2 ->event2 ...
+# Supported actions: drop_fields, drop_event, include_fields
+#filters:
+  #- drop_fields:
+  #   equals:
```

Comment:
Not sure if the indentation will be correct, if someone removes all 4 # here? I think one more space for the last 3 is needed?